resources/assets/js/components/Twitter.vue

```diff
@@ -74,14 +74,14 @@
         data() {
             return {
                 displayingTopTweetSince: new moment(),
-                onDisplay: [],
+                tweets: [],
                 waitingLine: [],
                 ownScreenName: '@spatie_be'
             };
         },
 
         created() {
-            this.onDisplay = JSON.parse(this.initialTweets).map(tweetProperties => new Tweet(tweetProperties));
+            this.tweets = JSON.parse(this.initialTweets).map(tweetProperties => new Tweet(tweetProperties));
```

Comment:
Could you look in to removing this `JSON.parse` call while we're at it? Vue should be able to do the parsing for us by declaring the property as a property that needs to be interpreted as JavaScript at the component implementation.

E.g.

```html
<twitter :initial-tweets='{"..":".."}'></twitter>
```